apps/web/vibes/soul/sections/featured-products-carousel/index.tsx

```diff
@@ -45,3 +45,31 @@ export function FeaturedProductsCarousel({ title, description, cta, products }:
     </section>
   )
 }
+
+export function FeaturedProductsCarouselSkeleton({
+  title,
+  description,
+  cta,
```

Comment:
Are these going to always be available? i.e., not streamable?

Reply:
In Catalyst, yes, in general, maybe not?